collaboration/README.md

```diff
@@ -1,5 +1,42 @@
 # Collaboration
-
 <!-- group norms summary -->
 
+**Summary**: Norms were agreed upon and will be used to maintain effective teamwork.
+
 <!-- group norms list -->
+## Our List of Group Norms
+
+### 💬 Communication
+
+- Communicate via WhatsApp and Slack channels
+- Respect the opinion of others and their way of working
+- Be patient with late responses
+
+### 🤝  Collaboration
+
+- Provide regular updates on important milestones in your work
+
+### 🔒 Accountability
+
+- Setting clear goals and expectations for the role and responsibilities of each
+- Setting deadlines for completing part of or all of deliverables
+
+### ⚖️ Conflict Resolution
+
+- Foster a culture of understanding and forgiveness
+- Offer constructive criticism but avoid personal attacks
+- If a conflict occurs, learn from it to strengthen dynamics and avoid future conflicts
+
+### 📆  Meetings
+
+- Reach a consensus for an agreed upon schedule and stick to it
+- Be well prepared
+
+### 👥 Behavioral
+
+- Respect diversity and inclusion
+- Avoid offensive language and behaviour
+- Maintain confidentiality when needed
+
+---
+Group Name: Code Collaborators
```

Comment:
Lastly
2. End of File: Add a single blank line at the end of the file to resolve the MD047/single-trailing-newline error.
thanks!